modules/distribution/src/repository/resources/dbscripts/apimgt/postgresql.sql

```diff
@@ -362,81 +364,81 @@ DROP TABLE IF EXISTS IDP;
-                  UNIQUE (IDP_ID, ROLE),
-                  FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
+			ID INTEGER DEFAULT NEXTVAL('IDP_ROLE_SEQ'),
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			ROLE VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, ROLE),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
 
 DROP TABLE IF EXISTS IDP_ROLE_MAPPING;
 DROP SEQUENCE IF EXISTS IDP_ROLE_MAPPING_SEQ;
 CREATE SEQUENCE IDP_ROLE_MAPPING_SEQ;
 CREATE TABLE IDP_ROLE_MAPPING (
-                  ID INTEGER DEFAULT NEXTVAL('IDP_ROLE_MAPPING_SEQ'),
-                  IDP_ROLE_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  USER_STORE_ID VARCHAR (253),
-                  LOCAL_ROLE VARCHAR(253),
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_ROLE_ID, TENANT_ID, USER_STORE_ID, LOCAL_ROLE),
-                  FOREIGN KEY (IDP_ROLE_ID) REFERENCES IDP_ROLE(ID) ON DELETE CASCADE);
+			ID INTEGER DEFAULT NEXTVAL('IDP_ROLE_MAPPING_SEQ'),
+			IDP_ROLE_ID INTEGER,
+			TENANT_ID INTEGER,
+			USER_STORE_ID VARCHAR (253),
+			LOCAL_ROLE VARCHAR(253),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ROLE_ID, TENANT_ID, USER_STORE_ID, LOCAL_ROLE),
+			FOREIGN KEY (IDP_ROLE_ID) REFERENCES IDP_ROLE(ID) ON DELETE CASCADE);
 
 DROP TABLE IF EXISTS IDP_CLAIM;
 DROP SEQUENCE IF EXISTS IDP_CLAIM_SEQ;
 CREATE SEQUENCE IDP_CLAIM_SEQ;
 CREATE TABLE IDP_CLAIM (
-                  ID INTEGER DEFAULT NEXTVAL('IDP_CLAIM_SEQ'),
-                  IDP_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  CLAIM VARCHAR(254),
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_ID, CLAIM),
-                  FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
+			ID INTEGER DEFAULT NEXTVAL('IDP_CLAIM_SEQ'),
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			CLAIM VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, CLAIM),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
 
 DROP TABLE IF EXISTS IDP_CLAIM_MAPPING;
 DROP SEQUENCE IF EXISTS IDP_CLAIM_MAPPING_SEQ;
 CREATE SEQUENCE IDP_CLAIM_MAPPING_SEQ;
 CREATE TABLE IDP_CLAIM_MAPPING (
-                  ID INTEGER DEFAULT NEXTVAL('IDP_CLAIM_MAPPING_SEQ'),
-                  IDP_CLAIM_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  LOCAL_CLAIM VARCHAR(253),
-                DEFAULT_VALUE VARCHAR(255),
-            IS_REQUESTED VARCHAR(128) DEFAULT '0',
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_CLAIM_ID, TENANT_ID, LOCAL_CLAIM),
-                  FOREIGN KEY (IDP_CLAIM_ID) REFERENCES IDP_CLAIM(ID) ON DELETE CASCADE);
+			ID INTEGER DEFAULT NEXTVAL('IDP_CLAIM_MAPPING_SEQ'),
+			IDP_CLAIM_ID INTEGER,
+			TENANT_ID INTEGER,
+			LOCAL_CLAIM VARCHAR(253),
+		    DEFAULT_VALUE VARCHAR(255),
+	    	IS_REQUESTED VARCHAR(128) DEFAULT '0',
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_CLAIM_ID, TENANT_ID, LOCAL_CLAIM),
+			FOREIGN KEY (IDP_CLAIM_ID) REFERENCES IDP_CLAIM(ID) ON DELETE CASCADE);
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04